src/modules/account/components/RemoveSelectedAccountFlow/RemoveSelectedAccountFlow.js

```diff
@@ -1,15 +1,12 @@
 import React from 'react';
 import { withRouter } from 'react-router';
-import { selectSearchParamValue } from 'src/utils/searchParams';
 import RemoveAccount from '../RemoveAccount/RemoveAccount';
-import { useAccounts } from '../../hooks';
+import { useCurrentAccount } from '../../hooks';
 
-const RemoveSelectedAccountFlow = ({ history }) => {
-  const address = selectSearchParamValue(history.location.search, 'address');
-  const { getAccountByAddress } = useAccounts();
-  const account = getAccountByAddress(address);
+const RemoveSelectedAccountFlow = () => {
```

Comment:
I am not sure if this can lead to a bug? it can happen that we are not on selected account while removing? we have manage accounts flow where we can remove multiple accounts, there current wont be set?